src/Apps/Artwork/ArtworkApp.tsx

```diff
@@ -292,6 +294,12 @@ const TrackingWrappedArtworkApp: React.FC<Props> = props => {
 
   const websocketEnabled = !!sale?.extendedBiddingIntervalMinutes
 
+  const initialAlertCriteria = {
+    attributionClass: compact([attributionClass?.internalID]),
+    artistIDs: compact(artists).map(artist => artist.internalID),
+    additionalGeneIDs: [mediumType?.filterGene?.slug as string],
```

Comment:
Could we use `compact` here as well to filter out `undefined` and `null` values instead of the type assertion `as string`?

Reply:
Ahh, that's a good point. Will update 👍🏼